### PR TITLE
edits

### PR DESCRIPTION
**Pull request by stanmak1991**

### CODE REVIEW

File: src/com/company/Task1.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.company;
+
+public class Task1 {
+    public static int totalizator(int a, int b, int x, int y){
```
Comment (Siril-O) [inline src/com/company/Task1.java]: Use descriptive variable names ex: firstTeamScore

File: src/com/company/Task1.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package com.company;
+
+public class Task1 {
```
Comment (Siril-O) [inline src/com/company/Task1.java]: Use descriptive class name ex: ScoreCalculator

File: src/com/company/Task1.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+    public static int totalizator(int a, int b, int x, int y){
+        int result = (a < b != x < y) || (a > b != x > y) || ((a != b) != (x != y))? 0 : ((a-b)==(x-y)) ? 2:1;
```
Comment (Siril-O) [inline src/com/company/Task1.java]: result variable can be removed

File: src/com/company/Task2.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+                if (y < left || y > right) { System.out.print(" "); }
+                else { System.out.print("*"); }
+            }
+            left--;
+            right++;
+            System.out.println();
+        }
+
+        left = 0;
+        right = arg*2-1;
```
Comment (Siril-O) [inline src/com/company/Task2.java]: Wrong formatting of spaces

File: src/com/company/Task2.java [hunk 5]
```diff
@@ -0,0 +1,31 @@
+package com.company;
+
+public class Task2 {
+    public static void rhomb(int arg){
```
Comment (Siril-O) [inline src/com/company/Task2.java]: method name should be a verb

File: src/com/company/Task3.java [hunk 6]
```diff
@@ -0,0 +1,11 @@
+package com.company;
+
+public class Task3 {
+    public static int PowModified(int a, int b) {
```
Comment (Siril-O) [inline src/com/company/Task3.java]: Why static ?

File: src/com/company/Task3.java [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+        int result = a;
```
Comment (Siril-O) [inline src/com/company/Task3.java]: what if a is -10 ?

File: src/com/company/Task2.java (hunk 5 above)
Comment (Siril-O) [reply src/com/company/Task2.java]: What if arg is  -10. Also variable name is invalid

File: src/com/company/Task4.java [hunk 8]
```diff
@@ -0,0 +1,15 @@
+package com.company;
+
+public class Task4 {
+    public static int fibonacci(int arg){
+        if (arg == 1 || arg == 2) {return 1;}
+        else { return fibonacci(arg-1)+fibonacci(arg-2); }
```
Comment (Siril-O) [inline src/com/company/Task4.java]: Add spaces, wrong formatting,  use Idea autoformat

File: src/com/company/Task5.java [hunk 9]
```diff
@@ -0,0 +1,176 @@
...
+        endTime = System.nanoTime();
+        sortingTime(startTime, endTime, "Selection Sort");
+
+        array = createBigArray();
+        startTime = System.nanoTime();
+        bubbleSort(array);
+        endTime = System.nanoTime();
+        sortingTime(startTime, endTime, "Bubble Sort");
+    }
+    public static int[] createBigArray() {
```
Comment (Siril-O) [inline src/com/company/Task5.java]: create class with descriptive name and make static methods as instance methods

File: src/com/company/Task5.java [hunk 10]
```diff
@@ -0,0 +1,176 @@
...
+        int[] array = new int[1000_000];
+        for (int i = 0; i < array.length; i++){
+            Random r = new Random();
+            array[i] = r.nextInt(1000_000);
```
Comment (Siril-O) [inline src/com/company/Task5.java]: Make all hardcode as constants

File: src/com/company/Task5.java [hunk 11]
```diff
@@ -0,0 +1,176 @@
...
+
+    public static void quickSort(int[] arr, int startIndex, int lastIndex) {
+        if (arr.length == 0) return;
+        int pointer = arr[lastIndex];
+        int i = startIndex, y = lastIndex;
+        while (i <= y) {
+            while (arr[i] < pointer && i <= lastIndex) ++i;
+            while (arr[y] > pointer && y >= startIndex) --y;
+            if (i <= y) {
+                int temp = arr[i];
```
Comment (Siril-O) [inline src/com/company/Task5.java]: Swap can be done as method to avoid it duplication

File: src/com/company/Task5.java [hunk 12]
```diff
@@ -0,0 +1,176 @@
...
+
+            return;
+        } else if (arr.length > 2) {
+            if ((arr.length % 2) == 1) b = new int[(arr.length / 2) + 1];
+            else if (arr.length % 2 == 0) b = new int[arr.length / 2];
+            else b = new int[arr.length];
+
+            for (int i = 0; i < b.length; i++) b[i] = arr[i];
+
+            c = new int[arr.length - b.length];
```
Comment (Siril-O) [inline src/com/company/Task5.java]: What does c name  means ?

File: src/com/company/Task5.java [hunk 13]
```diff
@@ -0,0 +1,176 @@
...
+                int temp = arr[i];
+                arr[i] = arr[y];
+                arr[y] = temp;
+                ++i; --y;
+            }
+        }
+        if (y > startIndex) quickSort(arr, startIndex, y);
+        if (i < lastIndex) quickSort(arr, i, lastIndex);
+    }
+    public static void mergeSort(int[] arr) {
```
Comment (Siril-O) [inline src/com/company/Task5.java]: Split this method to multiple smaller methods, it is hard to read it.

File: src/com/company/Task3.java (hunk 6 above)
Comment (stanmak1991) [reply src/com/company/Task3.java]: Because we don't need to create instance to use this method, f.e. class Math and method pow static too

**Pull request closed without merging**